packages/webcomponents/src/components/payout-details/payout-details-core.tsx

```diff
@@ -68,6 +72,15 @@ export class PayoutDetailsCore {
     });
   }
 
+  handleRecordClick = (id: string) => {
+    if (this.payout) {
```

Comment:
I think we could simplify the JSX a bit by moving that check into the handleRecordClick method itself.


```suggestion
    if (this.payout && this.enableRecordClick) {
```

Reply:
We can't because the `DetailItem` component is checking if the `onClick` prop is undefined or not to add the link styles.